rclrs/src/node/subscription.rs

```diff
@@ -125,6 +126,28 @@ where
         })
     }
 
+    /// Returns the name of the topic which is being subscribed
+    pub fn get_topic(&self) -> Result<String, RclrsError> {
+
+        let mut _topic_name: String = String::default();
+        // SAFETY: No preconditions for the function used
+        // Edge cases for all unsafe variables have been handled
+        // The unsafe variables created get dropped within the unsafe scope as soon as they're used
+        unsafe {
+            let raw_topic_pointer = rcl_subscription_get_topic_name((&mut *self.handle.lock()) as *mut _);
+            if let Ok(name) =  CStr::from_ptr(raw_topic_pointer).to_str() {
```

Comment:
Mostly a style preference, but I like the match syntax more than if..else.. for cases like this:

```rust
match CStr::from_ptr(raw_topic_pointer).to_str() {
  Ok(name) => name.to_string(),
  Err(_) => Err(RclsError::RclError{code: crate::error::RclReturnCode::TopicNameInvalid,
    msg: None,
    // Unable to give an error message as the String field in RclErrorMsg is private
  },
}
```